server/test/repository/QuestionRepositoryTest.java

```diff
@@ -299,4 +300,201 @@ public void createOrUpdateDraft_managesUniversalTagCorrectly()
     question = repo.createOrUpdateDraft(nextQuestionDefinition);
     assertThat(question.getQuestionTags().contains(QuestionTag.UNIVERSAL)).isFalse();
   }
+
+  @Test
+  public void createOrUpdateDraft_managesPrimaryApplicantInfoTagsCorrectl()
+      throws UnsupportedQuestionTypeException {
+    QuestionModel nameQuestion = testQuestionBank.applicantName();
+    QuestionModel dateQuestion = testQuestionBank.applicantDate();
+    QuestionModel emailQuestion = testQuestionBank.applicantEmail();
+    QuestionModel phoneQuestion = testQuestionBank.applicantPhone();
+    ImmutableSet<PrimaryApplicantInfoTag> nameTag =
```

Comment:
suggestion (non-blocking): Since you do this a few times, I wonder if it's worth writing a helper method that could give you back all of the tags for each `PrimaryApplicantInfoTag` wrapped in `ImmutableSet.of`. Although that might be more trouble than it's worth.